docs/docs/coverage/language/python.md

```diff
@@ -27,7 +27,7 @@ The following table provides an outline of the features Trivy offers.
 | pip             | requirements.txt |            -            |     Include      |                  -                   |    ✓     |                    ✓                     |
 | Pipenv          | Pipfile.lock     |            ✓            |     Include      |                  -                   |    ✓     |                Not needed                |
 | Poetry          | poetry.lock      |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
-| uv              | uv.lock          |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
+| uv              | uv.lock          |            ✓            |     Include      |                  ✓                   |    -     |                Not needed                |
```

Comment:
I think we should write the default behavior, like [Node.js](https://github.com/aquasecurity/trivy/blob/03647db96f36b2aba766f65fb0c213dbf937ae62/docs/docs/coverage/language/nodejs.md). Or am I missing something?

```suggestion
| uv              | uv.lock          |            ✓            |     Exclude      |                  ✓                   |    -     |                Not needed                |
```

Reply:
Fixed https://github.com/aquasecurity/trivy/pull/8134/commits/5d65e2f6c6dd6776667728a230866f1dd0dd14b5